charts/spire/charts/spire-server/templates/pre-upgrade-hook.yaml

```
@@ -73,7 +68,7 @@ spec:
         args:
           - patch
           - validatingwebhookconfiguration
-          - {{ $webhookname }}
+          - {{ .Release.Namespace }}-{{ include "spire-controller-manager.fullname" . }}-webhook
```

```suggestion
          - {{ printf "%s-%s-webhook" .Release.Namespace (include "spire-controller-manager.fullname" .) }}
```

Maybe use the same style in both places